3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -300,9 +300,12 @@ def initialize(debug, staging, enrich, shred, es, archive_raw, rdb_load, archive
           ].select { |el|
             is_cloudfront_log(collector_format) || is_ua_ndjson(collector_format)
           }
+          # uncompress events that are gzipped since this format is unsplittable and causes issues
+          # downstream in the spark enrich job snowplow/snowplow#3525
+          if collector_format == "clj-tomcat" then
```

Comment:
@alexanderdean in the end it's only clj-tomcat since cf and ndjson are lzo-consolidated above and thrift is already lzo.

Reply:
Makes sense